SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -107,16 +107,20 @@ extension HomeViewController: UICollectionViewDataSource, UICollectionViewDelega
             return cell ?? TestimonialsCollectionViewCell()
             }
         case lastestNewsCollectionView:
-            if indexPath.row == min(lastestNewsData.count, 4) {
+            if indexPath.row == min(sliderViewModel!.getNewsCount(), 4) {
                 let cell = lastestNewsCollectionView.dequeueReusableCell(withReuseIdentifier: "seeMoreCell", for: indexPath) as? SeeMoreCollectionViewCell
 
                 return cell ?? SeeMoreCollectionViewCell()
             } else {
             let cell = lastestNewsCollectionView.dequeueReusableCell(withReuseIdentifier: "newscell", for: indexPath) as? NewsCollectionViewCell
             
-            cell?.newsImage.image = lastestNewsData[indexPath.row].image
-            cell?.newsDescription.text = lastestNewsData[indexPath.row].epigraph
-            
+                let imagePath = self.sliderViewModel?.getNews(at: indexPath.row).image
+                let imageUrl = URL(string: imagePath!)
+                
```

Comment:
force unwrap could lead to crash,  it's enough that someone in made a mistake, NEVER force unwrap something coming form a server